tests/tx_test.go

```diff
@@ -39,14 +39,16 @@ func TestTxs(t *testing.T) {
 
 	var tcs []*testCase
 
-	tcs = append(tcs, makeBankTxs()...)
-	tcs = append(tcs, makeKycTxs()...)
-	tcs = append(tcs, makeMaintenaceTxs()...)
-	tcs = append(tcs, makeFeeTxs()...)
-	tcs = append(tcs, makeNonFungibleTokenTxs()...)
-	tcs = append(tcs, makeFungibleTokenTxs()...)
-	tcs = append(tcs, makeNameservicesTxs()...)
-	tcs = append(tcs, makeMultisigTxs()...)
+	// tcs = append(tcs, makeBankTxs()...)
```

Comment:
Please make sure these lines are not commented in your last commit.

Reply:
Hi, Mostafa

I already done house-keeping base on this, which committed:

https://github.com/maxonrow/maxonrow-go/commit/ac7a4232aa519dfcdc0152476399ff53be30f24b


Please review.